### PR TITLE
Keunwoo

### PR DESCRIPTION
**Pull request by kw9212**
- HW0B PR created (UNI: ks3651)

### CODE REVIEW

File: ks3651.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Open Source Development
+
+Name: KeunWoo Song
+
+UNI: ks3651
+
+Github: https://github.com/kw9212/project-proposals-s2023/tree/keunwoo
```
Comment (timkpaine) [inline ks3651.md:7]: link to your GitHub profile

File: ks3651.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Project Proposal
+
+As of now, I would like to contribute on one of the four open source projects below:
```
Comment (timkpaine) [inline ks3651.md:11]: these should be in the `Project Contributions` section, and in this section you should brainstorm the type of project you wish to create

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: ks3651.md (hunk 1 above)
Comment (kw9212) [reply ks3651.md]: Thank you for pointing this out, professor. I changed it.

File: ks3651.md (hunk 2 above)
Comment (kw9212) [reply ks3651.md]: Thank you for pointing this out, professor. I changed this too.

As a beginner with Git and GitHub, during my experimentation, I accidentally revised the contents of HW0(keunwoo branch) to what should have been assigned to HW1. I created separate markdown files for each project and contribution plan and submitted a pull request for HW1.

**Pull request closed without merging**